civis/io/_tables.py

```diff
@@ -626,7 +626,9 @@ def dataframe_to_civis(df, database, table, api_key=None, client=None,
 
     with TemporaryDirectory() as tmp_dir:
         tmp_path = os.path.join(tmp_dir, 'dataframe_to_civis.csv')
-        df.to_csv(tmp_path, encoding='utf-8', index=False, **kwargs)
+        to_csv_kwargs = {'encoding': 'utf-8', 'index': False}
+        to_csv_kwargs.update(kwargs)
+        df.to_csv(tmp_path, **to_csv_kwargs)
```

Comment:
@jkulzick, is there a reason to prevent users from changing `encoding` or `utf-8` here?

Reply:
I can't think of one, we should probably let the user override these